chia/full_node/mempool_manager.py

```diff
@@ -366,10 +362,9 @@ async def add_spend_bundle(
         """
 
         # Skip if already added
-        if spend_name in self.mempool.spends:
-            cost: Optional[uint64] = self.mempool.spends[spend_name].cost
-            assert cost is not None
-            return uint64(cost), MempoolInclusionStatus.SUCCESS, None
+        existing_item = self.mempool.get_spend_by_id(spend_name)
+        if existing_item is not None:
+            return uint64(existing_item.cost), MempoolInclusionStatus.SUCCESS, None
```

Comment:
Cost is already a `uint64` so I'd just `return existing_item.cost, MempoolInclusionStatus.SUCCESS, None`